drivers/sensor/adltc2990/adltc2990.c

```diff
@@ -141,9 +141,10 @@ static int adltc2990_trigger_measurement(const struct device *dev)
 	return i2c_reg_write_byte_dt(&cfg->bus, ADLTC2990_REG_TRIGGER, 0x1);
 }
 
-static int32_t adltc2990_fetch_property_value(const struct device *dev,
+static int adltc2990_fetch_property_value(const struct device *dev,
 					      enum adltc2990_monitoring_type type,
-					      enum adltc2990_monitor_pins pin)
+					  enum adltc2990_monitor_pins pin,
+					  int32_t *output)
```

Comment:
Hey @benjaminbjornsson, thanks for this rework. This was actually something i was planning to do 👍 .